statscache_plugins/volume/by_package.py

```diff
@@ -1,28 +1,29 @@
 import collections
 
 import statscache.plugins
+from statscache_plugins.volume.utils import VolumePluginMixin
```

Comment:
I think you forgot to commit this ``utils.py`` file.  Take a look in your working dir -- are there any others?